src/transformers/chai-should.test.js

```diff
@@ -23,7 +23,7 @@ function testChanged(msg, source, expectedOutput) {
 testChanged(
     'removes imports and does basic conversions of should and expect',
     `
-        var expect = require('chai').expect;
+        const { expect } = require('chai');
```

Comment:
Can we add another test verifying that `var expect = require('chai').expect;` still converts correctly.